src/midi/midi_mt32.cpp

```diff
@@ -356,9 +356,12 @@ MidiHandler_mt32::~MidiHandler_mt32()
 // ratio.
 void MidiHandler_mt32::SetMixerLevel(const AudioFrame &levels) noexcept
 {
-	const float gain = std::max(desired.left, desired.right);
-	if (service)
-		service->setOutputGain(gain);
+	const float gain = std::max(levels.left, levels.right);
+	{
```

Comment:
Every time I go back to this code, I find myself thinking: why is there a function inside a function here? ;)

Next time when placing a block of code like this, let's put an empty line before preceding code, to break this visual similarity - so code won't be in "shape" of function.

Not a problem for this PR of course - it's just me being pedantic :)

Reply:
Oh yes, now I see it as a function (And can't unsee it 😅)
Yes.. next time we touch this I'll space them out.